DQM/L1TMonitor/interface/L1TObjectsTiming.h

```diff
@@ -76,6 +76,15 @@ class L1TObjectsTiming : public DQMEDAnalyzer {
   std::vector<MonitorElement*> etsum_eta_phi_MHT;
   std::vector<MonitorElement*> etsum_eta_phi_MHTHF;
 
+  std::vector<MonitorElement*> den_muons_eta_phi;
```

Comment:
No need to make a vector of the denominator histograms. The denominator histogram will be filled with the sum of entries for all BX and one denominator can be used for all BX numerator histograms.